README.md

```diff
@@ -90,7 +90,7 @@ The installation prerequisites are the same as described in the TensorFlow
 
         git clone https://github.com/tensorflow/ngraph-bridge.git
         cd ngraph-bridge
-        git checkout master
+        git checkout v0.14.1
```

Comment:
is 0.14.1 our recommended version for bridge?

Reply:
hmm, i started making my changes on top v0.14.1. should i change this to `v0.15.0-rc1` or leave it `master`? @avijit-nervana

Reply:
We could leave out any changes in the version in this PR maybe. This is going to master right?

Reply:
yeah, this got automatically applied when i checked out to v0.14.1. i will remove this.